zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -50,7 +51,23 @@ def __call__(
 
 @mazepa.taskable_operation_cls
 @attrs.mutable
-class ReduceByWeightedSum:
+class ReduceOperation(ABC):
+    def __call__(
+        self,
+        src_idxs: List[VolumetricIndex],
+        src_layers: List[VolumetricBasedLayerProtocol],
+        red_idx: VolumetricIndex,
+        roi_idx: VolumetricIndex,
+        dst: VolumetricBasedLayerProtocol,
+        processing_blend_pad: Vec3D[int],
+        processing_blend_mode: Literal["linear", "quadratic"],
```

Comment:
This should be `Union[Literal["linear", "quadratic"], Literal["max"]]`

Reply:
It feels wrong to have the ABC have such concrete information about the classes that implement it